src/components/ledger/LedgerConnector.tsx

```diff
@@ -48,11 +50,13 @@ export interface LedgerConnectorData {
 interface LedgerConnectorProps {
   onSuccess: (data: LedgerConnectorData) => void;
   onTroubleshoot: () => void;
+  isWalletExist?: boolean;
 }
 
 export function LedgerConnector({
   onSuccess,
   onTroubleshoot,
+  isWalletExist,
```

Comment:
If I understand correctly, this prop tells `LedgerConnector` to verify if the secret already exists, right?

If so, I think we should rename it. Right now it sounds more as a statement than a question/request to check for something.

Maybe: `checkIfWalletExists` or `withWalletDeduplicationWarning` ?